assets/js/data/cart/actions.js

```diff
@@ -206,6 +206,34 @@ export function* removeCoupon( couponCode ) {
 	return true;
 }
 
+/**
+ * Adds an item to the cart:
+ * - Calls API to add item.
+ * - If successful, yields action to add item from store.
+ * - If error, yields action to store error.
+ * - Sets cart item as pending while API request is in progress.
+ *
+ * @param {number} productId Product ID to add to cart.
+ * @param {number} quantity Number of product ID being added to cart.
+ */
+export function* addItemToCart( productId, quantity = 1 ) {
+	try {
+		const { response } = yield apiFetchWithHeaders( {
+			path: `/wc/store/cart/add-item`,
+			method: 'POST',
+			data: {
+				id: productId,
```

Comment:
Should this API use a `cartItemKey` rather than `productId`, e.g. for consistency with the `remove-item` API?

Reply:
Doh, perhaps this is because we are making a new item, the new key will be returned :) (Ignore previous comment)